src/main/java/christmas/domain/Benefit.java

```diff
@@ -0,0 +1,46 @@
+package christmas.domain;
+
+import christmas.enumeration.BenefitType;
+
+import java.util.EnumMap;
+
+public class Benefit {
+    EnumMap<BenefitType, Integer> benefit;
```

Comment:
```suggestion
    EnumMap<BenefitType, Integer> benefits;
```
로 하는게 더 확실하게 이해될 것 같아요!

Reply:
저는 여기서 BenefitType과 Integer가 count를 가지고 있는지, price의 관계인지 바로 파악이 안되었어요. Integer를 따로 VO로 만들어보거나 변수 자체를 benefitPrice?로 하거나 해도 좋을 것 같아요.